src/core/postAttachments.ts

```diff
@@ -0,0 +1,52 @@
+import {tmpdir} from 'os';
+import {sep} from 'path';
+import joplin from 'api';
+import {Attachment} from '../model/attachment.model';
+import {AttachmentProperties} from '../model/attachmentProperties.mode';
```

Comment:
a typo? `mode` vs `model` on the previous line

Reply:
> a typo?
Yes, I fixed it.

Reply:
> a typo?
Yes, I fixed it.